src/commands/host_features.rs

```diff
@@ -5,7 +5,18 @@ use crate::{
 
 impl AdbTcpConnexion {
     /// Lists available ADB server features.
-    pub fn host_features(&mut self) -> Result<Vec<HostFeatures>> {
+    pub fn host_features<S: ToString + Clone>(
```

Comment:
Why do you force type to implement the `Clone` trait ?
This is not needed as S is not cloned but `to_string()'ed`